test/tools/test_tools_target.py

```diff
@@ -211,3 +276,59 @@ def test_channel_is_priv_level_oper_user():
     assert not channel.is_op(user.nick)
     assert not channel.is_halfop(user.nick)
     assert not channel.is_voiced(user.nick)
+
+
+def test_channel_clear_user():
+    channel = target.Channel(Identifier('#chan'))
+    brian = target.User(Identifier('Brian'), 'brian', 'example.com')
+
+    channel.add_user(brian, plugin.OP)
+    assert brian.nick in channel.users
+    assert channel.is_op(brian.nick)
```

Comment:
This strikes me as a separate test of the functionality of `is_op`, but it does match the shape of the test suite around it, so whatever I guess :grin:

Reply:
This test assume `is_op` is already well tested and work as advertised. :smile: